### ISSUE TITLE
Remove ice skates

### ISSUE THREAD

**Issue by lionleaf** [ninjadev/globalgamejam2016#199]
Sweet movement code when we were circles, but with feet I think it's time to take the skates off :)

**Comment by iver56** [comment]
For reference: The icy breaking has been tweaked here: https://github.com/ninjadev/globalgamejam2016/commit/bfde0cd0c75b11e14d21c3959b3c1305f60ec30a

**Comment by lionleaf** [comment]
Nice; that increases the friction on the ice, I want to see how it feels without Ice at all (ie. no sideways motion at all). The ice might actually be good for hiding latency, though, so it's worth testing for a few games.

**Comment by iver56** [comment]
Yeah, you may increase the friction a little more, but I wouldn't recommend immediate stopping, because I suspect that it would feel kind of unnatural/rough. Most games have some deceleration before the character stops.

**Comment by sigvef** [comment]
I agree, some deceleration is nice.
